components/IframePreviewCard/provider.tsx

```diff
@@ -112,7 +117,8 @@ export const IframePreviewCardProvider = ({ children }) => {
   }, [setPreviewCardEnabled, width])
 
   useEffect(() => {
-    const handleOpenPreview = async (e) => {
+    // TODO: TS support
+    const handleOpenPreview = async (e: any) => {
```

Comment:
`React.MouseEvent<HTMLElement, MouseEvent>`？

Reply:
Line 123 `const url = parseHref(target.href)`
Line 126 `const openTarget = target.getAttribute('target')`
Line 128 `window.open(target.href, openTarget)`
Line 130 `window.location.href = target.href`
it can't work